contracts/Vault.vy

```diff
@@ -1117,6 +1117,13 @@ def updateStrategyPerformanceFee(
     log StrategyUpdatePerformanceFee(strategy, performanceFee)
 
 
+@internal
+def _revokeStrategy(strategy: address):
```

Comment:
I couldn't find a way to call an external method so I made the implementation `internal`